backend/app/repository/gerrit_provider.py

```
@@ -0,0 +1,928 @@
+        """
+        if response_text.startswith(")]}'"):
+            return response_text[4:]
+        return response_text
+
+    def _make_request(
+        self,
+        method: str,
+        url: str,
+        username: str,
+        http_password: str,
+        params: Dict[str, Any] = None,
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)